bundles/EcommerceFrameworkBundle/Controller/PricingController.php

```diff
@@ -191,6 +192,108 @@ public function deleteAction(Request $request)
         return $this->adminJson($return);
     }
 
+    /**
+     * @Route("/copy", methods={"POST"})
```

Comment:
Should this implement JS Routing (#6146)?

Reply:
not yet implemented in ecommerce framework